src/helix-ui/styles/modules/stepper.less

```diff
@@ -0,0 +1,100 @@
+/* ======================================= *\
+ * BETA BETA BETA BETA BETA BETA BETA BETA *
+\* ======================================= */
+hx-accordion.beta-hxStepper {
+  counter-reset: step;
+
+  hx-accordion-panel {
+    border-top: 1px solid @gray-300;
+
+    &:last-child {
+      border-bottom: 1px solid @gray-300;
+    }
+
+    header[slot="header"] {
+      display: flex;
+      padding: 1rem;
+
+      > * {
+        display: flex;
+        align-items: center;
+      }
+
+      // TODO: clarify mouse hover/click behavior with Design
+      &:hover {
+        .beta-hxStepCounter {
+          background-color: @cyan-500;
+          color: @gray-0;
+        }
+      }
+    }
+
+    &[open] { // NOTE: no spec!
+      .beta-hxStepCounter {
+        background-color: @cyan-500;
+        color: @gray-0;
```

Comment:
Discussion for later around the progress state of the hxStepCounter having value